Base/usr/share/man/man5/GML/Usage.md

```diff
@@ -6,43 +6,47 @@ GML Usage
 
 How to use GML in SerenityOS C++ applications
 
+## Note
+
+This manpage describes the new method of loading GML files via generated C++ code. There also is the runtime `load_from_gml` function which has the same behavior. This method should not be used except for specific use cases such as [GMLPlayground](help://man/1/Applications/GMLPlayground).
+
 ## CMake
 
-Include `compile_gml()` your applications CMake file. The header file name and GML string name are not fixed but must follow this convention.
+Include `compile_gml()` your application's CMake file. The generated source file name is not fixed, but should follow this convention.
 
 ```cmake
-compile_gml(MyApp.gml MyAppGML.h my_app_gml)
+compile_gml(MyApp.gml MyAppGML.cpp)
 ```
 
-Include the name of the header file that will be compiled from your GML file in your `SOURCES`.
+Include the name of the source file that will be compiled from your GML file in your `SOURCES`.
 
 ```cmake
 set(SOURCES
-    MyAppGML.h
+    MyAppGML.cpp
 )
 ```
 
 ## C++
 
-You can then reference the variable you set (e.g. `my_app_gml`) in your main C++ file using `load_from_gml()`.
+The C++ source file that is generated will provide an implementation for the `ErrorOr<NonnullRefPtr<App::Widget>> App::Widget::try_create()` function, given that the root widget of the GML file is `App::Widget`. For this to work, you have to add a declaration for this function in the header of the `App::Widget` class. (The function will not collide with functions provided by `Core::Object`, do not worry.) Additionally, there must be a public no-argument constructor in `App::Widget`, which is used by the `try_create` implementation.
```

Comment:
The part about the public no-argument constructor doesn't seem accurate anymore. The generated `try_create` method is in the correct namespace to use private constructors, and the Help widget that you converted even makes use of that.